index.html

```diff
@@ -349,6 +349,23 @@ <h2>About Myself</h2>
 
 
 
+      </section>
+      <section class="contact-form">
+        <h2>Contact Me</h2>
+        <p>If you have an application you are interested in developing, a
+          feature that you need built or a project that needs coding.
+          I’d love to help with it!</p>
+        <form action="/contact" method="post" enctype="multipart/form-data">
+          <input type="text" placeholder="Full names"
+            max="25" min="10"
+            required="required">
```

Comment:
- [ ] You dont have to add the value `required` to your input field. Your input field just requires a key called required. No need to add a value to the attribute 

e.g 

     ` <input type="text" placeholder="Lorem Ipsum" maxLength="30" required >`